debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/VirtualThreadsCommand.java

```diff
@@ -49,6 +51,31 @@
  *     ...
  */
 public class VirtualThreadsCommand extends Command {
+	private static J9ObjectFieldOffset vthreadOffset = null;
+	private static J9ObjectFieldOffset vmRefOffset = null;
+	private static J9ObjectFieldOffset nameOffset = null;
+
+	private static J9ObjectFieldOffset getVthreadOffset(J9ObjectPointer continuation) throws CorruptDataException {
+		if (vthreadOffset == null) {
+			vthreadOffset = J9ObjectHelper.getFieldOffset(continuation, "vthread", "Ljava/lang/Thread;");
+		}
+		return vthreadOffset;
+	}
```

Comment:
When I suggested helper methods, my thinking was that they would return the field of interest rather than just its offset, for example, instead of this method we would have:
```java
	private static J9ObjectPointer getVirtualThread(J9ObjectPointer continuation) throws CorruptDataException {
		if (vthreadOffset == null) {
			vthreadOffset = J9ObjectHelper.getFieldOffset(continuation, "vthread", "Ljava/lang/Thread;");
		}
		return J9ObjectHelper.getObjectField(continuation, vthreadOffset);
	}
```
The offset isn't otherwise used at the call site.